locale/dbr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,8639 @@
+
+msgid "Dictionaries & Language Packs"
+msgstr "sʞɔɐԀ ǝƃɐnƃuɐ⅂ ⅋ sǝıɹɐuoıʇɔıᗡ"
+
+msgid "Add-ons"
+msgstr "suo-pp∀"
+
+#. L10n: relative time in the past, like '4 days ago'
+msgid "{0} ago"
+msgstr "oƃɐ {0}"
+
+msgid "Not allowed"
+msgstr "pǝʍoʅʅɐ ʇoN"
+
+msgid "<h1>Oops! Not allowed.</h1> <p>You tried to do something that you weren't allowed to.</p>"
+msgstr "<p>.oʇ pǝʍoʅʅɐ ʇ,uǝɹǝʍ noʎ ʇɐɥʇ ƃuıɥʇǝɯos op oʇ pǝıɹʇ no⅄</p> <h1>.pǝʍoʅʅɐ ʇoN ¡sdoO</h1>"
+
+msgid "<p>Try going back to the previous page, refreshing and then trying again.</p>"
+msgstr "<p>.uıɐƃɐ ƃuıʎɹʇ uǝɥʇ puɐ ƃuıɥsǝɹɟǝɹ ´ǝƃɐd snoıʌǝɹd ǝɥʇ oʇ ʞɔɐq ƃuıoƃ ʎɹ⊥</p>"
+
+msgid "Page Not Found"
+msgstr "punoℲ ʇoN ǝƃɐԀ"
+
+msgid "Firefox Add-ons"
+msgstr "suo-pp∀ xoɟǝɹıℲ"
+
+msgid "Sorry, but we can't find anything at the address you entered. If you followed a link to an add-on, it's possible that add-on has been removed by its author."
+msgstr ".ɹoɥʇnɐ sʇı ʎq pǝʌoɯǝɹ uǝǝq sɐɥ uo-ppɐ ʇɐɥʇ ǝʅqıssod s,ʇı ´uo-ppɐ uɐ oʇ ʞuıʅ ɐ pǝʍoʅʅoɟ noʎ ɟI .pǝɹǝʇuǝ noʎ ssǝɹppɐ ǝɥʇ ʇɐ ƃuıɥʇʎuɐ puıɟ ʇ,uɐɔ ǝʍ ʇnq ´ʎɹɹoS"
+
+msgid "Suggested Pages"
+msgstr "sǝƃɐԀ pǝʇsǝƃƃnS"
+
+msgid "Browse all extensions"
+msgstr "suoısuǝʇxǝ ʅʅɐ ǝsʍoɹԐ"
+
+msgid "Browse all themes"
+msgstr "sǝɯǝɥʇ ʅʅɐ ǝsʍoɹԐ"
+
+msgid "Add-ons Home Page"
+msgstr "ǝƃɐԀ ǝɯoH suo-pp∀"
+
+#, python-format
+msgid "If you followed a link from somewhere, please %(open_bug_link)sfile an issue%(close_bug_link)s. Tell us where you came from and what you were looking for, and we'll do our best to fix it."
+msgstr ".ʇı xıɟ oʇ ʇsǝq ɹno op ʅʅ,ǝʍ puɐ ´ɹoɟ ƃuıʞooʅ ǝɹǝʍ noʎ ʇɐɥʍ puɐ ɯoɹɟ ǝɯɐɔ noʎ ǝɹǝɥʍ sn ʅʅǝ⊥ .%(close_bug_link)sǝnssı uɐ ǝʅıɟ%(open_bug_link)s ǝsɐǝʅd ´ǝɹǝɥʍǝɯos ɯoɹɟ ʞuıʅ ɐ pǝʍoʅʅoɟ noʎ ɟI"
+
+msgid "Not Found"
+msgstr "punoℲ ʇoN"
+
+#, python-format
+msgid ""
+"<h1>We're sorry, but we can't find what you're looking for.</h1> <p> The page or file you requested wasn't found on our site. It's possible that you clicked a link that's out of date, or typed in "
+"the address incorrectly. </p> <ul> <li>If you typed in the address, please double check the spelling.</li> <li> If you followed a link from somewhere, please <a href=\"https://github.com/mozilla/"
+"addons-server/issues/new/\">file an issue</a>. Tell us where you came from and what you were looking for, and we'll do our best to fix it. </li> </ul> <p>Or you can just jump over to some of the "
+"popular pages on our website.</p> <ul> <li>Are you interested in a <a href=\"%(rec)s\">list of featured add-ons</a>?</li> <li> Do you want to <a href=\"%(search)s\">search for add-ons</a>? You may "
+"go to the <a href=\"%(search)s\">search page</a> or just use the search field above. </li> <li> If you prefer to start over, just go to the <a href=\"%(home)s\">add-ons front page</a>. </li> </ul>"
+msgstr ""
+"<ul> <li> .<a href=\"%(home)s\">ǝƃɐd ʇuoɹɟ suo-ppɐ</a> ǝɥʇ oʇ oƃ ʇsnɾ ´ɹǝʌo ʇɹɐʇs oʇ ɹǝɟǝɹd noʎ ɟI </li> <li> .ǝʌoqɐ pʅǝıɟ ɥɔɹɐǝs ǝɥʇ ǝsn ʇsnɾ ɹo <a href=\"%(search)s\">ǝƃɐd ɥɔɹɐǝs</a> ǝɥʇ oʇ oƃ "
```

Comment:
This is an example of what could be handled better, it would be nice if only the child elements with text were re-ordered. Without having the H1 at the end. It should maybe only reverse the children if there's a text-node + html mix. I'll think about how to handle that.